src/common/ConfUtils.cc

```diff
@@ -403,12 +403,15 @@ process_line(int line_no, const char *line, std::deque<std::string> *errors)
     ACCEPT_UNQUOTED_VAL,
     ACCEPT_QUOTED_VAL,
     ACCEPT_COMMENT_START,
+    ACCEPT_QUOTE_COMMENT_START,
```

Comment:
could you help outline the state-machine using ascii art in a comment? as `process_line()` is difficult to understand for new developers now.

Reply:
will update the state machine comment latter

Reply:
added the state-machine  ascii art in a comment. Please take a review.